RecoBTag/MXNet/python/pfParticleNetDiscriminatorsJetTags_cfi.py

```diff
@@ -0,0 +1,99 @@
+import FWCore.ParameterSet.Config as cms
+
+pfParticleNetDiscriminatorsJetTags = cms.EDProducer(
+   'BTagProbabilityToDiscriminator',
+   discriminators = cms.VPSet(
+      cms.PSet(
+         name = cms.string('TvsQCD'),
+         numerator = cms.VInputTag(
+            cms.InputTag('pfParticleNetJetTags', 'probTbcq'),
+            cms.InputTag('pfParticleNetJetTags', 'probTbqq'),
+            ),
+         denominator = cms.VInputTag(
+            cms.InputTag('pfParticleNetJetTags', 'probTbcq'),
+            cms.InputTag('pfParticleNetJetTags', 'probTbqq'),
+            cms.InputTag('pfParticleNetJetTags', 'probQCDbb'),
+            cms.InputTag('pfParticleNetJetTags', 'probQCDcc'),
+            cms.InputTag('pfParticleNetJetTags', 'probQCDb'),
+            cms.InputTag('pfParticleNetJetTags', 'probQCDc'),
+            cms.InputTag('pfParticleNetJetTags', 'probQCDothers'),
```

Comment:
[to consider for a follow up PR to simplify these very verbose patterns]
it would be nice to write some simple config tool so that it can make a complete PSet based on name, numeratorList and denominatorList and the jetTagsModuleName. It would create a PSet with a call to smth like
```python
qcdAnyList = ["QCDbb", "QCDcc", ... ]
...
probabilityPSet(name="TvsQCD", module = "pfParticleNetJetTags", numeratorList = ["Tbcq", "Tbqq"], denominatorList= [numeratorList, qcdAnyList]
```

Reply:
To be done in a follow-up PR.

Reply:
~ OK for a follow up PR here; mostly due to already existing coding patterns.
However, reusing of qcdAnyList (or similarly named) list of input tags may be already helpful here as well (applies to pfMassDecorrelatedParticleNetDiscriminatorsJetTags_cfi.py as well); it seems easy to implement as well.

Reply:
I will do it in a separate PR to clean up this and DeepAK8 together.